packages/twenty-server/src/modules/messaging/message-import-manager/drivers/imap/services/imap-find-sent-mailbox.service.ts

```diff
@@ -0,0 +1,167 @@
+import { Injectable, Logger } from '@nestjs/common';
+
+import { isNumber } from 'class-validator';
+import { ListResponse, type ImapFlow } from 'imapflow';
+
+/**
+ * Service to find sent folder using IMAP special-use flags
+ *
+ * This service uses IMAP special-use extension (RFC 6154) to identify
+ * the sent folder by looking for the \Sent flag rather than relying on
+ * folder names which can vary across providers and locales.
+ *
+ * Falls back to regex-based detection if special-use flags are not available.
+ * The regex pattern is inspired by imapsync's comprehensive folder mapping.
+ */
+@Injectable()
+export class ImapFindSentMailboxService {
+  private readonly logger = new Logger(ImapFindSentMailboxService.name);
+
+  public async findSentMailbox(client: ImapFlow): Promise<string | null> {
```

Comment:
findSentMailboxName

Reply:
Settled on folder, its consistent with `messageFolders` I also renamed it across all the imap files